admin/app/jobs/FastlyCloudwatchLoadJob.scala

```diff
@@ -10,21 +13,50 @@ object FastlyCloudwatchLoadJob extends ExecutionContexts with Logging {
   // Samples in CloudWatch are additive so we want to limit duplicate reporting.
   // We do not want to corrupt the past either, so set a default value (the most
   // recent 15 minutes of results are unstable).
+
+  // This key is (service, name, region)
   val latestTimestampsSent = mutable.Map[(String, String, String), Long]().
     withDefaultValue(DateTime.now().minusMinutes(15).getMillis())
 
+  // Be very explicit about which metrics we want. It is not necessary to cloudwatch everything.
+  val allFastlyMetrics: List[SamplerMetric] = List(
```

Comment:
You could simplify this to make it a bit more DRY, but not a massive issue, nor a blocker.